libs/json-api-nestjs/src/lib/mixin/pipes/index.ts

```diff
@@ -95,10 +96,6 @@ export function bodyRelationshipPatchPipeMixin(): PipeMixin {
   return BodyRelationshipPatchPipe;
 }
 
-export function idPipeMixin(
-  entity: Entity,
-  connectionName: string,
-  config: ConfigParam
-): PipeMixin {
-  return config.pipeForId;
+export function idPipeMixin(entity: Entity, connectionName: string): PipeMixin {
+  return factoryMixin(entity, connectionName, IdSchemaPipe);
```

Comment:
I think will be better use config, by default use **ParseIntPipe**, by if need use other pipe, provide **ParseUUIDPipe**. 
```
import { ParseUUIDPipe } from '@nestjs/common';
....
JsonApiModule.forRoot({
....,
options: {
pipeForId: ParseUUIDPipe
}
})
```

What do you think?

Reply:
That would be clean, indeed.

I only worry if there's some use-case to mix and match between `SEQ` and `UUID` at the same time. Like I'm testing right now with both generation strategies.

Personally I wouldn't mind if we make it a config parameter like you suggested, though. I'll give it a try over the weekend.